backend/solver/origuide/tools/plot.py

```diff
@@ -79,7 +79,7 @@ def plot3d(vertices: List[Vertex], edges: List[Edge], faces: List[Face], forces)
     if CONFIG['DEBUG_PLOT_VELOCITIES']:
         ax.quiver(xs, ys, zs, vel_xs, vel_ys, vel_zs, color='r')
 
-    box_lim = 16
+    box_lim = 2
```

Comment:
This value is strictly related to `BOUNDING_BOX_DIAG_LEN`, maybe this should be: `BOUNDING_BOX_DIAG_LEN` optionally divided by `sqrt(3)`